### ISSUE TITLE
`proc-module = no` in config file, but pixbuf_proc.so helper still gets used

### ISSUE THREAD

**Issue by Flurrywinde** [mk-fg/infinite-image-scroller#5]
I output the current config with `./infinite-image-scroller.py --conf-dump` and put it here: `~/.config/infinite-image-scroller.ini`.

The helper is being used when I run it, as proven by:
```
2020-12-24 18:04:54 :: DEBUG :: Adding image: images/60c8e1e14a5585a2520410c81f64cf82.png
2020-12-24 18:04:54 :: DEBUG :: pixbuf_proc [queue]: images/60c8e1e14a5585a2520410c81f64cf82.png
2020-12-24 18:04:54 :: DEBUG :: pixbuf_proc [thread]: images/60c8e1e14a5585a2520410c81f64cf82.png
2020-12-24 18:04:54 :: DEBUG :: pixbuf_proc [signal]: images/60c8e1e14a5585a2520410c81f64cf82.png
2020-12-24 18:04:56 :: DEBUG :: Adding image: images/602575.06pf3ajv82z0w.png
2020-12-24 18:04:56 :: DEBUG :: pixbuf_proc [queue]: images/602575.06pf3ajv82z0w.png
2020-12-24 18:04:56 :: DEBUG :: pixbuf_proc [thread]: images/602575.06pf3ajv82z0w.png
2020-12-24 18:04:56 :: DEBUG :: pixbuf_proc [signal]: images/602575.06pf3ajv82z0w.png
```
So `--conf-dump` isn't outputting the right thing it seems?

Then, when I run `./infinite-image-scroller.py images/ --debug -l -a1:.01`, the config is used (I know because I changed `box-spacing` to 0, and it did it.), but the helper is still used.

Sorry, maybe the code to process this part of the config just hasn't been added yet, but I thought I'd report these findings just in case.

**Comment by mk-fg** [comment]
Oh, it's just a bug, didn't realize that it was being accidentally exposed as a configurable value.
Fixed in dd7fbb6 by adding a underscore prefix to it, thanks for reporting it.

Dunno if it's needed as an option, as there shouldn't really be any downside to using .so module if it's available, and not using it is presumably easy enough too - just rm it :)

**Comment by Flurrywinde** [comment]
Yeah, I agree it seems best just to remove it.
